build.gradle

```diff
@@ -209,7 +215,10 @@ dependencies {
   testCompile group: 'org.junit.platform', name: 'junit-platform-commons', version: versions.junitPlatform
   testCompile group: 'org.junit.platform', name: 'junit-platform-engine', version: versions.junitPlatform
 
-  testCompile group: 'org.springframework.boot', name: 'spring-boot-starter-test'
+  testCompile group: 'org.springframework.boot', name: 'spring-boot-starter-test', withoutJunit4
+  testCompile group: 'com.github.tomakehurst', name: 'wiremock-jre8', version: '2.25.0', withoutJunit4
```

Comment:
I believe presence of wiremock indicates that test using it is already integration test

Reply:
it will be used in integration test.

Reply:
hence wrongly assigned - in `test` scope, not `integration`